### DISCUSSION TITLE
Store server config on shared NFS storage instead of /var/lib/pufferpanel/server directory

### DISCUSSION THREAD

**Discussion by ervu** [pufferpanel/pufferpanel#1300] [General]
Is it possible to use shared NFS shares to store server configs instead of local /var/lib/pufferpanel/server directory? This is helpfull in a large enviroment with multiple nodes but also to do storage based snapshots/backup server configs. Have tried to modify a server binding during creating, to add new storage mapping but that didnt work out. Love to hear the possibilities...

![Screenshot 2024-02-11 220706](https://github.com/pufferpanel/pufferpanel/assets/69848862/edb34f9d-41da-452b-ad23-e7b55c1bd3b5)

**Comment by LordRalex** [comment]
You should mount the share to /var/lib/pufferpanel/server

The bindings that are there are meant to add folders to the server. That does not change where the server is stored at.

**Reply by ervu** [reply]
Good idea.... i have tested this and it is working great!
What is the purpose for the bindings?

**Reply by LordRalex** [reply]
The idea for the bindings would be if you had an extra folder on your actual server that you wanted the container to have, such as like a cache. Most of the time, it's not used, but it's just exposed anyways.

**Reply by ervu** [reply]
Hosting the servers on a central NFS is great voor DR reasons as PP doesn't have backup module build in. With the file management option or SFTP it can cover a basic backup , but it would be nice to have a build-in backup or snapshot feature. Are there any backup plugins available?
